demos/input/button/source/PlayState.hx

```diff
@@ -0,0 +1,23 @@
+package;
+
+import flixel.ui.FlxButton;
+import flixel.FlxG;
+import flixel.FlxState;
+import flixel.util.FlxColor;
+
+class PlayState extends FlxState
+{
+	var button:FlxButton;
```

Comment:
I don't like to put variables on class scope unless necessary (this could be a local variable instead).